docs/howtos/create_custom_appcharts.md

```diff
@@ -156,3 +156,24 @@ $ epinio app chart show fluent
 
 If the new Helm chart is not correctly registered in Epinio, check that the
 `RELEASE_NAMESPACE` was properly set in `fluentd-appchart.yaml`.
+
+## Going further
+
+The standard app chart, as well as the chart created here use a kubernetes `Deployment` as
+the main resource describing the active application.
```

Comment:
```suggestion
Both the standard and custom app charts use a kubernetes `Deployment` as
the main resource describing the active application.
```

Reply:
The new phrase makes it sound as if all custom app charts will use a `Deployment`. In light of the change allowing more than `Deployment` I wanted to call out the tutorial's custom chart specifically, instead saying it generally, as the new phrasing. A slight foreshadowing of the next sentence where the restriction is explained and lifted.

Am I making sense ?

Reply:
yes, so please discard my comment